2016/Day 2-1.py

```diff
@@ -0,0 +1,14 @@
+sides = {"L": [0, -1], "U": [-1, 0], "R": [0, 1], "D": [1, 0]}
+keypad = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
+with open("Day 2 - input") as f:
+    x, y = 1, 1
+    code = ""
+    for line in f:
+        ll = 0
+        for direction in line[:-1]:
```

Comment:
Use the strip method instead of slicing. It more clearly explains that you want to remove the newline character. Unless it is for some other reason. In that case please explain